src/gui/screen_printing.cpp

```diff
@@ -91,6 +95,7 @@ void screen_printing_data_t::stopAction() {
     }
     switch (GetState()) {
     case printing_state_t::PRINTED:
+        change_print_state();
```

Comment:
`change_print_state()` (with `Screens::Access()->Close();`) could cause extra unwanted redraw just before screen is closed.

Please remove it.